Makefile

```diff
@@ -11,15 +11,23 @@ DEPLOY_METERING_PKG := $(GO_PKG)/cmd/deploy-metering
 # gofmt
 VERIFY_FILE_PATHS := cmd pkg test manifests
 
-DOCKER_BUILD_CMD = docker build
+# Prefer podman, if not on the path use docker.  Suppress errors so we don't see any
+# potentially misleading messages like "which: no podman in ..." in the output.
+CONTAINER_RUNTIME_CMD = podman
+ifeq (, $(shell which podman 2> /dev/null))
+    CONTAINER_RUNTIME_CMD = docker
+endif
```

Comment:
Can we just remove this block and instead just default `CONTAINER_RUNTIME_CMD = docker` with the intention that you would override it when running a make target, e.g. `make verify-docker CONTAINER_RUNTIME_CMD=podman`